examples/internal/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -2272,6 +2160,120 @@
         ]
       }
     },
+    "/v1/example/a_bit_of_everything/{uuid}": {
+      "get": {
```

Comment:
Note - `get` and `put` switched order in the generated swagger.json, as I didn't add the `path_param_name` override to `grpc.gateway.examples.internal.proto.sub2.IdMessage` (the request message in used by the `get` operation), and then introduced a new path only for some of the operations. This makes it a bit more difficult to compare this file.